entr.c

```diff
@@ -400,10 +400,17 @@ run_utility(char *argv[]) {
 	char **new_argv;
 	char *p, *arg_buf;
 	int argc;
+	int stdin_pipe[2];
 
-	if (restart_opt == 1)
+	if (restart_opt == 1) {
 		terminate_utility();
 
+		ret = pipe(stdin_pipe);
+		if (ret != 0)
```

Comment:
Since we don't use `ret` anywhere, let's inline the test `if (pipe(stdin_pipe) != 0)`

Reply:
sure